Automattic/wp-calypso
# Reader: update reader sidebar icons

Related to https://github.com/Automattic/wp-calypso/pull/95887

This updates the Reader sidebar to use WordPress icons

Ref: pe7F0s-2gw-p2

## Proposed Changes

* Update icons in Reader sidebar

| **Before** | **After** |
|--------|-------|
|  <img width="284" alt="Screenshot 2024-10-31 at 15 31 51" src="https://github.com/user-attachments/assets/df648ece-71bb-415d-b04f-cb380e1dad6b">  | <img width="283" alt="Screenshot 2024-10-31 at 15 31 07" src="https://github.com/user-attachments/assets/adff7095-cd4d-44a6-84b6-76cd93e7901d">  |

## Why are these changes being made?


* Tidy up UX

## Testing Instructions



* Go to calypso.live site `/read` and confirm the sidebar menu items look and behave ok.

## Pre-merge Checklist
